features-sdk/support/world.js

```diff
@@ -40,20 +41,26 @@ class KuzzleWorld {
     return this._protocol;
   }
 
-  parseObject (dataTable) {
-    if (typeof dataTable.rowsHash !== 'function') {
-      throw new Error('Argument is not a datatTable');
-    }
+  parseObject(dataTable) {
+    const rawContent = dataTable.rowsHash();
+    const content = {};
 
-    const content = dataTable.rowsHash();
+    for (const [path, value] of Object.entries(rawContent)) {
+      if (value.includes('_AGO_')) {
+        // format: "_5m_AGO_"
+        const timeAgo = ms(value.split('_')[1]);
 
-    for (const key of Object.keys(content)) {
-      content[key] = JSON.parse(content[key]);
+        _.set(content, path, this.props.now - timeAgo);
+      }
+      else {
+        _.set(content, path, eval(`var o = ${value}; o`));
```

Comment:
eval can be harmful

Reply:
It's only for the tests so it's not dangerous for our clients